.azure/pipelines/ci.yml

```diff
@@ -81,6 +81,8 @@ variables:
     value: /bl:artifacts/log/Release/Build.Installers.binlog
   - name: WindowsArm64InstallersLogArgs
     value: /bl:artifacts/log/Release/Build.Installers.Arm64.binlog
+  - name: EnableRichCodeNavigation
+    value: 'true'
```

Comment:
Please revert changes to our regular CI. There's an error in the file now in any case.

Reply:
Will do - I never intended to merge these changes, just made these changes to get a glimpse of what the time to run would be and to verify the rich nav pipeline would run (I can't push my branch to the upstream source so I can't make a new pipeline off my branch until after it's merged... it makes me have to do some juggling to validate confidently that things will work as expected 😃 )